src/java/Controller/PDFinfo.java

```diff
@@ -78,142 +78,142 @@ public String getEmployeeName(String empID) throws Exception{
         String str = njdbc.queryForObject(SQL, namedParams, String.class);
         return str;
     }
-	
-	/**
+    
+    /**
     * Get stream ID and name of streams employee has undertaken
     * @param empID - the employee id of the employee
     * @return a list of strings, formatted as [id1, name1, id2, name2, ...]
     * @throws java.lang.Exception
     */
     public ArrayList<String> getStreamIDName(String empID) throws Exception{
-		ArrayList<String> list = new ArrayList();
+        ArrayList<String> list = new ArrayList();
         Statement s1 = c1.createStatement();
      
-		ResultSet r1=s1.executeQuery("select s.stream_id, s.stream_name from Stream s, Class c, Employees e where s.stream_id=c.stream_id and c.class_id=e.class_id and e.employee_id='"+empID+"'");
-		while(r1.next()){
-			list.add(r1.getString(1));
-			list.add(r1.getString(2));
-		}
+        ResultSet r1=s1.executeQuery("select s.stream_id, s.stream_name from Stream s, Class c, Employees e where s.stream_id=c.stream_id and c.class_id=e.class_id and e.employee_id='"+empID+"'");
+        while(r1.next()){
+            list.add(r1.getString(1));
+            list.add(r1.getString(2));
+        }
               
-		r1.close();
+        r1.close();
         s1.close();
 
         return list;
-	}
-	
+    }
+    
     /**
     * Get overall average score of an employee
     * @param empID - the employee id of the employee
     * @return a string representation of the average score
     * @throws java.lang.Exception
     */
-	public String getAverageScoresByEmployeeID(String empID) throws Exception{
+    public String getAverageScoresByEmployeeID(String empID) throws Exception{
         String SQL = "select avg(e.scores) from Employees_take_Modules e, Modules m where m.module_id=e.module_id and e.employee_id= :id";
         SqlParameterSource namedParams= new MapSqlParameterSource("id",empID);
         String str = njdbc.queryForObject(SQL, namedParams, String.class);
 
-		return str;
-	}
-	
+        return str;
+    }
+    
     /**
     * Get average foundation score of an employee
     * @param empID - the employee id of the employee
     * @return a string representation of the average score
     * @throws java.lang.Exception
     */
-	public String getAverageScoresByFoundationEmployeeID(String empID) throws Exception{
-        String SQL = "select avg(e.scores) from Employees_take_Modules e, Modules m where m.category='Foundation' and m.module_id=e.module_id and e.employee_id= :id";
+    public String getAverageScoresByFoundationEmployeeID(String empID) throws Exception{
+        String SQL = "select avg(e.scores) from Employees_take_Modules e, Modules m where m.category_id='FOUND01' and m.module_id=e.module_id and e.employee_id= :id";

```

Comment:
Looks like a lot of changes, but apparently we just can't all agree to use spaces instead or tabs, or at least have them autoexpand :P

These are ok changes, but be advised that I *am* refactoring the entire file to reflect that there may be more than the enumerated categories.